### PR TITLE
Feature page

### PR DESCRIPTION
**Pull request by Lichapa**
###  New York Times clone: article
In this project we:

- Added Navbar

- Added Images and video

- added sidebars using aside

- positioned element using grid and flex

- floated elements using float property

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+
+> This project consists of building an HTML document that matches the appearance of New york times [Article](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0).
+
+![screenshot](./images/Times.PNG)
+
+
+## Project Requirements
+- The project is meant to demonstrate 
+- position and floating of elements.
```
Comment (SafaErden) [inline README.md]: - [ ] It would be nice to make all the first letters capital.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,781 @@
...
+            </ul>
+        </div>
+        <div>
+
+        </div>
+
+        <div class="footer-bottom">
+
+            <a href="# ">Access more of The Times by creating a free account or logging in.</a>
+        </div>
```
Comment (SafaErden) [inline index.html]: - [ ]  The expandable footer should cover the full width and it also causes a horizontal scroll. Please make it covers the full with to create the same UI with the original page.

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,781 @@
...
+                                But Where Is That?</a>
+                            <br>
+                            <span class="gray very-small-text">Aug. 23</span>
+                        </p>
+                    </div>
+                </div>
+            </section>
+        </div>
+    </main>
+    <footer class="footer">
```
Comment (SafaErden) [inline index.html]: - [ ]  The main footer should also be centered. Please kindly fix it also.

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,781 @@
...
+                            <figcaption class="gray very-small-text ">
+                                <a href="#"></a>NASA/JPL-Caltech/University of Arizona</figcaption>
+                        </figure>
+                        <p><a href="#">How NASA Found the Ideal Hole on Mars to Land In</a>
+                            <br>
+                            <span class="gray very-small-text">July. 30</span>
+                        </p>
+                    </div>
+                </div>
+            </section>
```
Comment (SafaErden) [inline index.html]: - [ ] Also the bottom section is not centered as it is on the original page. Please fix it.

image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,781 @@
...
+                </p>
+                <p>
+                    Dr. Kovac and his collaborators, working in an experiment known as Bicep, for Background Imaging of
+                    Cosmic Extragalactic Polarization, reported their results in a scientific briefing at the Center for
+                    Astrophysics here on Monday and in a set of papers
+                    submitted to The Astrophysical Journal.
+                </p>
+            </article>
+        </section>
+        <section class="inflation">
```
Comment (SafaErden) [inline index.html]: - [ ] The original page's main section contains centered text and images. But on your page, some content is not centered. Please kindly fix it.

image

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,539 @@
+body {
```
Comment (SafaErden) [inline css/style.css:1]: - [ ] Make sure that basic CSS bad practices are not used ([article for support ](https://speckyboy.com/good-bad-css-practices)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

- So please apply DRY CSS rules.

**Review by SafaErden — CHANGES_REQUESTED** [review]
## Status: Require changes ♻️

**Hi TEAM, your project is awesome and it really looks exactly like the original. 👐 However, we need a little fix before we can proceed. Kindly find comments attached below**
##
**NOTES**

I would suggest you to use a code-formatter to prevent redundant empty lines and ugly indentations. 

##

**SUMMARY**

As I mentioned above, your project is almost done. You have some minor issues to solve. Please be an optimist and do not forget that the main purpose is to learn and practice. Do not let the requested changes discourage you 👍

##

*Kindly make the changes and submit another code review when you are ready*

Questions or comments? Reach me on [Twitter](https://twitter.com/safaerden) or [LinkedIn](https://www.linkedin.com/in/safaerden/) 🗨️

Happy coding_ 👨‍💻 

**Safa ERDEN**

File: index.html (hunk 4 above)
Comment (Lichapa) [reply index.html]: This has been resolved

**Review by voscarmv — APPROVED** [review]
# Impressive work, team! 🖥️ 👀 ✨ 

- Excellent work applying the changes required by the previous TSE.

# Project status: Approved ✨ 😎👍 💯 ✔️
### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**